app/ide-desktop/client/tests/creatingNewProject.spec.ts

```diff
@@ -0,0 +1,13 @@
+/** @file A test for basic flow of the application: open project and see if nodes appear. */
```

Comment:
nit re: file name - for test files named after actions, i name them like methods - so present (present simple) tense instead of continuous tense (`createNewProject`)